File: DashboardDemo/DashboardSample/Dashboard/ColumnElement.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using Telerik.WinControls.UI;
using System.Drawing;
using System.Windows.Forms;

namespace TicketTest
{
    public class ColumnElement: StackLayoutElement
    {
        ColumnHeaderElement header;
        StackLayoutElement content;

        public ColumnHeaderElement Header { get { return this.header; } }
        public StackLayoutElement Content { get { return this.content; } }

        public ColumnElement(string title)        
        {
            Header.Text = title;
        }

        protected override void InitializeFields()
        {
            base.InitializeFields();

            this.DrawFill = false;
            this.DrawBorder = true;
            this.BorderBoxStyle = Telerik.WinControls.BorderBoxStyle.FourBorders;
            this.BorderLeftWidth = 0;
            this.BorderTopWidth = 0;
            this.BorderRightWidth = 1;
            this.BorderRightColor = Color.Gray;
            this.StretchHorizontally = true;
            this.StretchVertically = true;
            this.Orientation = System.Windows.Forms.Orientation.Vertical;
        }

        protected override void CreateChildElements()
        {
            base.CreateChildElements();

            header = new ColumnHeaderElement();
            this.Children.Add(header);

            content = new StackLayoutElement();
            content.Orientation = Orientation.Vertical;
            content.StretchHorizontally = true;
            content.StretchVertically = true;
            content.AllowDrop = true;
            this.Children.Add(content);
        }
    }
}

File: DashboardDemo/DashboardSampleVB/Dashboard/ColumnElement.vb
Imports Telerik.WinControls.UI

Public Class ColumnElement
    Inherits StackLayoutElement

    Private _header As ColumnHeaderElement
    Private _content As StackLayoutElement

    Public ReadOnly Property Header As ColumnHeaderElement
        Get
            Return Me._header
        End Get
    End Property

    Public ReadOnly Property Content As StackLayoutElement
        Get
            Return Me._content
        End Get
    End Property

    Public Sub New(ByVal title As String)
        header.Text = title
    End Sub

    Protected Overrides Sub InitializeFields()
        MyBase.InitializeFields()
        Me.DrawFill = False
        Me.DrawBorder = True
        Me.BorderBoxStyle = Telerik.WinControls.BorderBoxStyle.FourBorders
        Me.BorderLeftWidth = 0
        Me.BorderTopWidth = 0
        Me.BorderRightWidth = 1
        Me.BorderRightColor = Color.Gray
        Me.StretchHorizontally = True
        Me.StretchVertically = True
        Me.Orientation = System.Windows.Forms.Orientation.Vertical
    End Sub

    Protected Overrides Sub CreateChildElements()
        MyBase.CreateChildElements()
        _header = New ColumnHeaderElement()
        Me.Children.Add(_header)
        _content = New StackLayoutElement()
        _content.Orientation = Orientation.Vertical
        _content.StretchHorizontally = True
        _content.StretchVertically = True
        _content.AllowDrop = True
        Me.Children.Add(content)
    End Sub
End Class